_posts/2025-05-04-acl2809-rit-bus-app.md

```diff
@@ -0,0 +1,43 @@
+# Ritchie's Bus Schedule: Eager loading for the welcome image
```

Comment:
You seem to be missing the jekyll "front matter" meta information - make sure you look at the source code for the sample blog post and follow that as a template

Reply:
without this, jekyll wont pick this page up as a blog post or render the markdown into HTML for you